notes-webapp/src/main/webapp/skin/less/notes/notes.less

```diff
@@ -309,11 +309,22 @@
     min-height: calc(~"100vh - 170px");
 
     .notes-application-header {
+      .note-actions-menu {
+        right: 35px;
+        top: 50px !important;
+        left: auto !important;
```

Comment:
Could you consider the RTL display here ?

Reply:
Done